content/assets/js/ie10-viewport-bug-workaround.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * IE10 viewport hack for Surface/desktop Windows 8 bug
+ * Copyright 2014-2017 The Bootstrap Authors
+ * Copyright 2014-2017 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+// See the Getting Started docs for more information:
+// https://getbootstrap.com/getting-started/#support-ie10-width
+
+(function () {
+  'use strict'
+
+  if (navigator.userAgent.match(/IEMobile\/10\.0/)) {
```

Comment:
I would suggest that for the MVP you make sure it works with one browser and only write code that will work for that. In the future if you want to expand you can verify the GUI on multiple browsers.